apps/arena/lib/arena/application.ex

```diff
@@ -15,6 +15,7 @@ defmodule Arena.Application do
       {Finch, name: Arena.Finch},
       # Start game launcher genserver
       Arena.GameLauncher,
+      Arena.BountyUpdater,
```

Comment:
```suggestion
      Arena.GameBountiesUpdater,
```
What do you think? Everything in-game is called GameSomething.

Or maybe just Bounties or BountyQuests. Not mandatory but I think the plural is more right to use in this case.

Reply:
Yeah, I'm not entirely convinced of the name I chose, I think I like GameBoundyCache, what do you think?